### PR TITLE
Capstone Project Build 1

### PR DESCRIPTION
**Pull request by pulgosodog**
Changes made acording to the **HTML/CSS & JavaScript capstone project - Conference page** milestones and requirements following the [guideline design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

1. Added to pages: indexhtml (The home page) & about.html (The about page) accessible through "More of Plantasia".
2. Pages made mobile first with two versions each made with media queries.
3. Implemented a pop up mobile menu for both pages using javascript.
4. The section "Available plants"(Featured speakers) HTML was made dinamically from objects from and with javascript 
5. Deployed page on GitHub pages https://pulgosodog.github.io/capstone-project/

### CODE REVIEW

**Review by taldr27 — COMMENTED** [review]
# Status: Invalidated 🔴
Hi @pulgosodog, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
You have submitted a project, that doesn't have the proper linters set up. This milestone requires you to use the `html-css-js` linter which is a new setup than the one you've been using so please visit the [linters config here](https://github.com/microverseinc/linters-config/tree/master/html-css-js) and update your repo accordingly.

After you've set up the linters you should see 5 checks (like the one below) instead of 4 checks.

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready. 🙏

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit. gif_

File: index.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        <button id="quit">
+        </button>
+        <ul id="menu-items" class="">
+            <li>
+                <a href="index.html"><img src="images/Plantasia-logo-inner-background.png" alt="Plantasia logo word"></a>
+            </li>
+            <li>
+                <a href="#intro">ABOUT</a>
+            </li>
+            <li>
```
Comment (uwadonat) [inline index.html:28]: - [ ] For now we can access `About` Page using `More of Placantia` link but it would be better to link this page also to `About` tab.

Comment (uwadonat) [reply index.html:28]: - [ ] As requested in the project requirements, please deploy the application using `github pages` and the link to your readme file.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @pulgosodog ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### Highlights
- Your commit messages are very descriptive ✔️
- You implemented well JavaScript best practices ✔️
- Amazing design ✔️
try

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (pulgosodog) [reply index.html:28]: Hello @uwadonat, regarding the first comment, can you rephrase it? I do not understand what you mean. You mean, change the href of the <a> tag with the value ABOUT to direct me to the about.html file (That is what About tab meant?)?
Note that the about <a> directs the user to the intro section, also, will it be ok having two links? What do I do with More Of plantasia link? Do I leave it blank?

Comment (pulgosodog) [reply index.html:28]: Done! Changed the `<a href="#intro">ABOUT</a>` to `<a href="html/about.html">ABOUT</a>`

Comment (pulgosodog) [reply index.html:28]: Updated the README.md file to have the live demo link on the corresponding section.

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @pulgosodog  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Made changes as requested by the previous reviewer :+1: 

### You met most of the requirements however I urge you to make the changes I marked as optional so that your project looks more professional :+1: 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                </h2>
+                <h4>
+                    @ 24 9th Ave, New York, NY 10014
+                </h4>
+            </div>
+
+        </section>
+    </div>
+
+    <section id="amenities">
```
[the remark this answers is not in the document]
Comment (pulgosodog) [reply index.html:117]: I appreciate the feedback 🙌, I'm going to add it in a future itineratiron bot as for now, I'm going to merge.

File: html/about.html [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    <title>Mother Earth's Plantasia</title>
+</head>
+
+<body>
+    <div id="menu-background" class="">
+        <button id="quit">
+        </button>
+        <ul id="menu-items" class="">
+            <li>
+                <a href="index.html"><img src="../images/Plantasia-logo-inner-background.png" alt="Plantasia logo word"></a>
```
[the remark this answers is not in the document]
Comment (pulgosodog) [reply html/about.html:23]: I'm going to link it in a future itinaration. Thanks! 🙌
